graphql_compiler/compiler/expressions.py

```diff
@@ -691,6 +691,7 @@ def to_match(self):
         # The MATCH versions of some operators require an inverted order of arguments.
         # pylint: disable=unused-variable
         regular_operator_format = '(%(left)s %(operator)s %(right)s)'
+        intersects_operator_format = '(%(operator)s(%(left)s, %(right)s).asList().size() > 0)'
```

Comment:
Please add this below the regular/inverted formats -- those two are related to each other, whereas this one is entirely separate and should not be between them.